homeassistant/components/analytics/analytics.py

```diff
@@ -138,6 +141,7 @@ async def send_analytics(self, _=None) -> None:
 
         if self.supervisor:
             supervisor_info = hassio.get_supervisor_info(self.hass)
+            operating_system_info = hassio.get_os_info(self.hass)
```

Comment:
This can raise `HassioAPIError`, right? Do we catch that somewhere?

Reply:
Nope, this just fetches the cached values
https://github.com/home-assistant/core/blob/a67b9eff174c79e628e7d1f5b8275c8652cad81b/homeassistant/components/hassio/__init__.py#L301-L308

Reply:
Right, `hassio` is the module not the handler instance. :+1: